src/App.js

```diff
@@ -1,13 +1,48 @@
 import React from "react";
-
+import Heading from "./Heading.js";
+import TouristInfoCards from "./TouristInfoCards.js";
 import Bookings from "./Bookings";
+import Footer from "./Footer.js";
 import "./App.css";
 
+const cities = [
+  {
+    name: "Manchester",
+    website: "visitmanchester.com",
```

Comment:
```suggestion
    website: "https://visitmanchester.com",
```

Reply:
Without the full url here then the link doesn't take me to the right site - in fact, it just links me back to the same page.